modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc

```diff
@@ -76,4 +77,24 @@ Deploy a transformation with multiple environment variables:
 rpk transform deploy --var FOO=BAR --var FIZZ=BUZZ
 ----
 
+Configure compression for batches output by data transforms. The default setting is "no compression," but you can choose from the following options:

```

Comment:
Is "no compression" the same as `none` listed below?

Reply:
Yes. I'll update